mesonbuild/modules/java.py

```diff
@@ -88,13 +88,13 @@ def generate_native_header(self, state: ModuleState, args: T.Tuple[T.Union[str,
 
     @FeatureNew('java.generate_native_headers', '0.62.0')
     @typed_pos_args('java.generate_native_headers',
-        varargs=(str, mesonlib.File, Target, CustomTargetIndex, GeneratedList))
+                    varargs=(str, mesonlib.File, Target, CustomTargetIndex, GeneratedList))
     @typed_kwargs('java.generate_native_headers',
-        KwargInfo('classes', ContainerTypeInfo(list, str), default=[], listify=True,
-            required=True),
-        KwargInfo('package', str, default=None))
+                  KwargInfo('classes', ContainerTypeInfo(list, str), default=[], listify=True,
+                            required=True),
+                  KwargInfo('package', str, default=None))
```

Comment:
Can you actually avoid indenting KwargInfo more than 4 spaces here? What we do elsewhere is break the first string argument out and put it on the next line.

Instead of
```python
@typed_pos_args('this is the name of something',
                KwargInfo('another name', args='bla')
                KwargInfo('moar names', args='blah',
                          otherargs='bleep bloop'))
```

Do:
```python
@typed_pos_args(
    'this is the name of something',
    KwargInfo('another name', args='bla'),
    KwargInfo(
        'moar names',
        args='blah',
        otherargs='bleep bloop',
    ),
)
```

Reply:
Of course! Changed push up now